CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+###  v0.1.0 - 07/08/2021: First development release 
+
+ - Initial development release
```

Comment:
```suggestion
### v0.1.0 - 07/08/2021
 - Initial development release
```
I think we should remove "First development release here" since it isn't conventional to give releases titles. It also doesn't add much, since the version number and description are used to indicate the same information.